pkg/rancher-desktop/components/RdSelect.vue

```diff
@@ -27,6 +27,13 @@ export default Vue.extend({
         this.$emit('input', newValue);
       },
     },
+    computedListeners(): Record<string, Function | Function[]> & { input: (e: any) => any; } {
```

Comment:
It may be a good idea to have (a more concise version of?) the PR description as a comment here, to explain why we're doing this.

Or maybe hook up the singular listener on `mount` or something?